pyscriptjs/src/main.ts

```diff
@@ -173,34 +163,23 @@ export class PyScriptApp {
         logger.info('config loaded:\n' + JSON.stringify(this.config, null, 2));
     }
 
-    // lifecycle (4)
-    async loadInterpreter() {
-        logger.info('Initializing interpreter');
-        if (this.config.interpreters.length == 0) {
-            throw new UserError(ErrorCode.BAD_CONFIG, 'Fatal error: config.interpreter is empty');
-        }
-
-        if (this.config.interpreters.length > 1) {
-            showWarning('Multiple interpreters are not supported yet.<br />Only the first will be used', 'html');
-        }
-
-        const interpreter_cfg = this.config.interpreters[0];
+    _get_base_url(): string {
+        const elem = document.currentScript as HTMLScriptElement;
```

Comment:
Note that this requires that pyscript is loaded via a `<script>` tag. One could imagine allowing it to be loaded via an ES6 module import in the future, but it's not important for now.

Reply:
added a comment in 673d25d